calico_containers/calico_ctl/profile.py

```diff
@@ -20,8 +20,8 @@
   calicoctl profile <PROFILE> tag (add|remove) <TAG>
   calicoctl profile <PROFILE> rule add (inbound|outbound) [--at=<POSITION>]
     (allow|deny) [(
-      (tcp|udp) [(from [(ports <SRCPORTS>)] [(tag <SRCTAG>)] [<SRCCIDR>])]
-                [(to   [(ports <DSTPORTS>)] [(tag <DSTTAG>)] [<DSTCIDR>])] |
+      (tcp|udp) [(from [(ports <SRCPORTS>)] [(tag <SRCTAG>)] [(cidr <SRCCIDR>)])]
```

Comment:
I've updated each `<SRCCIRD>, <DSTCIDR>` reference to have the `cidr` keyword now, and I've added the `cidr` keyword to the pretty print function.

I've briefly looked at doc opt and can't say for sure how easy it would be to modify for this.  I imagine it wouldn't be too difficult.

There was some other discussion I had with @spikecurtis about having our own parsing function that checks for the keywords and values types of the parameters passed in.  
This would be useful to us to avoid these issues and also to be able to allow different combinations of each rule (such as allowing ports by themselves, but not allowing a tag to be programmed without a cidr, etc.)  Spike was unsure of what Felix actually should allow, which is another thing to look into.

I think this current fix is usable and works in the meantime.  It may be worth talking about a bit more to decide if this should be the fix or if it should change now/be put on the backlog to improve later.